src/app/lib/config/services/thai.js

```diff
@@ -179,23 +181,27 @@ const service = {
         },
         {
           href: 'https://www.bbc.com/thai/institutional-38403477',
-          text: 'เงื่อนไขการใช้เว็บไซต์',
+          text: 'เงื่อนไขการใช้งานของ BBC',
+        },
+        {
+          href: 'https://www.bbc.com/thai/institutional-38403476',
+          text: 'เกี่ยวกับบีบีซี',
         },
         {
           href: 'https://www.bbc.com/thai/institutional-38497681',
           text: 'นโยบายความเป็นส่วนตัว',
         },
         {
           href: 'https://www.bbc.com/usingthebbc/cookies/',
-          text: 'Cookies',
+          text: 'คุกกี้ี้',
         },
         {
           href: 'https://www.bbc.com/thai/institutional-37981748',
-          text: 'ติดต่อบีบีซี',
+          text: 'ติดต่อบีบีซีี',
```

Comment:
Remove the ghostly 'o' https://github.com/bbc/simorgh/issues/3171

```suggestion
          text: 'ติดต่อบีบีซี',
```